mace/cli/fine_tuning_select.py

```diff
@@ -136,12 +159,13 @@ def parse_args() -> argparse.Namespace:
         default=1.0,
     )
     parser.add_argument("--seed", help="random seed", type=int, default=42)
-    return parser.parse_args()
+    parser.add_argument("--atomic_numbers", help="atomic numbers to keep for filtering", nargs="+", type=int, required=False)
```

Comment:
Here this seems like a sensible name for the option, but if it's also going to be in `mace_run_train`, _there_ it needs a more explicit name.  Maybe `--pt_filter_atomic_numbers` ?

Reply:
@ilyes319 any thoughts on name?